assets/icons/fitview.svg

```diff
@@ -1,3 +1,3 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 30">
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 30" fill="#d8d8d8">
```

Comment:
For these, in creator we can do something like `.react-flow__controls-button > svg`